jobs/config.json

```diff
@@ -1216,6 +1258,48 @@
       "sig-gcp"
     ]
   },
+  "ci-kubernetes-e2e-cos-gke-k8sbeta": {
+    "args": [
+      "--check-leaked-resources",
+      "--cluster=",
+      "--deployment=gke",
+      "--extract=ci/k8s-beta",
+      "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
+      "--gcp-node-image=gci",
+      "--gcp-zone=us-central1-f",
+      "--ginkgo-parallel",
+      "--gke-create-command=container clusters create --quiet --enable-kubernetes-alpha",
```

Comment:
You enable kubernetes alpha for all clusters.

Reply:
How about `ci-kubernetes-e2e-cos-gke-alpha-cluster-k8sbeta`?